recipes-support/curl/curl/0011-Excluding-HAVE_WINSOCK2_H-and-HAVE_WS2TCPIP_H-on-Pha.patch

```diff
@@ -1,27 +1,23 @@
-From 3f9f3f19bc3c9dbfd0bb9267c30fd7587ddac845 Mon Sep 17 00:00:00 2001
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Haris Okanovic <haris.okanovic@ni.com>
```

Comment:
Could you update the attribution info, since you modified the .patch contents.

Reply:
I thought this situation generally got solved by amending the commit, not scratching out the author. Would you take that instead?

Reply:
Yep; an amend would be fine. Thanks.

Reply:
Amended.